src/MassTransit/Transports/BaseReceiveContext.cs

```diff
@@ -115,6 +119,21 @@ public virtual Task NotifyFaulted(Exception exception)
         public Uri InputAddress { get; protected set; }
         public ContentType ContentType => _contentType.Value;
 
+        public Task Complete()
```

Comment:
Let's make these explicit, okay? Just so they don't pollute the auto-complete space.